client/src/components/Jobs/Jobs.scss

```
@@ -239,3 +239,12 @@
     }
   }
 }
+
+.no-results-message {
+  flex-grow: 1;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding-top: 200px;
+}
```

I think `flex-grow` and `align-items` don't do much here, since the parent element is `display: inline`
```suggestion
.no-results-message {
  display: flex;
  justify-content: center;
  padding-top: 200px;
}
```